.cirrus.yml

```diff
@@ -155,6 +155,16 @@ meta_task:
     clone_script: &noop mkdir -p $CIRRUS_WORKING_DIR  # source not needed
     script: /usr/local/bin/entrypoint.sh
 
+fedora36_build_task:
+    alias: fedora35_build
```

Comment:
YIKES! Um, no!

Reply:
this is what happens when you ask for changes! :), fixed